packages/blade/src/tokens/global/fontFamily.native.ts

```diff
@@ -1,6 +1,7 @@
 import { Platform } from 'react-native';
+import type { FontFamily } from './fontFamily.d';
```

Comment:
our tools only substitutes `.js`, `.ts`, `.native.ts` so now if I add `.d.ts` to resolution map in our tools it'll get confused and might pick the first match which will again be `fontFamily.native.ts` or `fontFamily.web.ts` so it might conflict. So added it explicitly to mean definitions